packages/ui-library/src/elements/tooltip/index.js

```diff
@@ -15,34 +15,29 @@ const positionClassNameMap = {
  * @param {string|JSX.Element} [as] Base component.
  * @param {string} [className] CSS class.
  * @param {string} [position] Position of the tooltip.
- * @param {boolean} isVisible Default state.
  * @returns {JSX.Element} Tooltip component.
  */
 
-const Tooltip = forwardRef( ( { id, children, as: Component, className, isVisible, position, ...props }, ref ) => {
+const Tooltip = forwardRef( ( { id, children, as: Component, className, position, ...props }, ref ) => {
 	return (
-		<>
-			{ isVisible && (
-				<Component
-					ref={ ref }
-					className={ classNames( "yst-tooltip",
-						positionClassNameMap[ position ],
-						className,
-					) }
-					role="tooltip"
-					id={ id }
-					{ ...props }
-				>
-					{ children || null }
-				</Component>
+		<Component
+			ref={ ref }
+			className={ classNames( "yst-tooltip",
+				positionClassNameMap[ position ],
+				className,
 			) }
-		</>
+			role="tooltip"
+			id={ id }
```

Comment:
If you're just passing the ID through without interacting with it. You can remove it from the specification and let it be indirectly through `props`.
For the story you might want to still specify it to show the `aria-describedby`, or add it to the `argTypes` so you get the interface for it

Reply:
Checked 👍